features/common.feature

```diff
@@ -39,10 +39,10 @@ Feature: Show common information for tooling
           "attributeConstraint": "CombinationUnique",
           "isRequired": false,
           "isSearchable": false,
-          "inputHint": ""
-        }
+          "inputHint": "SingleLine"
+        },
     """
-    When I run `product-type-update --projectKey import-101-64 --source product-type-boo-txt-num.json`
+    When I run `product-type-update --projectKey nicola --source product-type-boo-txt-num.json`
```

Comment:
because travis had configured env variables for nicola project - thats why there were failing tests